fe/estimator.py

```diff
@@ -163,7 +163,7 @@ def _deltaG(model, sys_params) -> Tuple[Tuple[float, List], np.array]:
     futures = []
     for lamb in model.lambda_schedule:
         args = (lamb, model.box, model.x0, model.v0, bound_potentials, model.integrator, model.equil_steps, model.prod_steps, model.barostat)
-        kwargs = {}  # Unused for now
+        kwargs = {"x_interval": 0}  # Unused for now
```

Comment:
Remove, was hitting memory limits on longer simulations thanks due to the default being 1000. Perhaps we should default to a proportion of the prod steps?

Reply:
Alternative is to strip out waters, but then again we may want to keep the waters in case we want to debug sampling problems.